src/css/skills.scss

```diff
@@ -0,0 +1,100 @@
+@import './core/constants';
+@import './core/mixins';
+
+//skill container
+$skill-s: 145px;
+$skill-m: 217.5px;
+$skill-l: 290px;
+$skill-xl: 435px;
+
+h2 {
+  font-size: $font-xl;
+}
+
+#skills {
+  height: max-content;
```

Comment:
be careful with this `max-content`, it is not supported by Edge: https://developer.mozilla.org/en-US/docs/Web/CSS/max-height#Browser_Compatibility